ddtrace/debugging/_expressions.py

```diff
@@ -183,9 +183,31 @@ def _compile_direct_operation(ast):
             raise ValueError("Invalid argument: %r" % arg)
         return [Instr("LOAD_CONST", len)] + value + [Instr("CALL_FUNCTION", 1)]
 
+    if _type == "ref":
+        if not isinstance(arg, str):
+            return None
```

Comment:
Whilst it makes sense for now to support just `str`, perhaps we could just try to compile `arg` as a `value_source`, in case in the future we'd want to offer some reflection-like capabilities. WDYT?

Reply:
LOVE THE IDEA.

Two thoughts, I don't see why ref op would need it -> do you have an example in mind?
Index op already doing it.
Maybe _compile_identifier could have this feature. so we can do dynamic GET_ATTR based on value source.
However, not sure how the DI syntax would look like, as "[]" is for index. so obj[some_var_as_attribute] will be translated to BIN_SUBSCR. 

So, I think leave it be for now. WDYT?

Reply:
> So, I think leave it be for now. WDYT?

👍  I don't think this is a blocker. This would just be a slight deviation from the stated syntax, since the argument is supposed to be a generic value and not just a literal string. The use case is if someone wanted to access a field dynamically, like it's done in Python with `getattr(obj, <dynamically computed attribute name>)`, but we can always add support for this later if we need to.

Reply:
yup. I think this also a good syntax: getattr(obj, var) - so for python developer it would be natural. then maybe the UI can even change it to `{"getmember":[{ref:"obj"},{"ref:":"var"}]}`. or even make it generic and have getmember(obj,field) as a method in EL.